src/api/inSightApiClient.js

```diff
@@ -0,0 +1,27 @@
+import request from 'request';
+import Weather from '../models/weather';
+
+export default function callInSight() {
+  const API_KEY = 'XFI8MAWpQ5sKZiB3Wvw5TaUCSaxxieSkt54RmAuJ';
```

Comment:
Hm, `master` has a `src/api/apiKey.js` file which isn't present in your checkout for some reason... I suggest recreating that file and using it as your key since the key is going to be shared between two API Clients.

(_see comment above_)